auth-rest-api/src/main/java/com/wipro/ats/bdre/md/auth/rest/AuthenticatorAPI.java

```diff
@@ -152,9 +155,9 @@ AuthResult validateToken(@PathVariable("authtoken") String token) {
     }
 
     @RequestMapping(value = "/admin/logout/{authtoken}", method = RequestMethod.DELETE)
-    public
+
     @ResponseBody
-    RestWrapper logoutByToken(@PathVariable("authtoken") String token, HttpSession session) throws LoginException {
+    public RestWrapper logoutByToken(@PathVariable("authtoken") String token, HttpSession session) throws LoginException {
         RestWrapper restWrapper = null;
         if (token != null) {
             try {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1166)